src/components/OneText.vue

```diff
@@ -66,6 +66,7 @@ export default {
           const tempstr = { time: 0, type: "F" };
           buslst.push(tempstr);
         }
+        this.buslst = buslst
```

Comment:
What does this code do? Explanation needed...!

Reply:
In my opinion, buslst would have value of null when util.isHalt() != 'halted'...

Reply:
`buslst` that have made by method while halt checking, need to apply to the view part that `this.buslst` do.
If without this line, the default value would be shown, {time :0, type:} (yes, because type has no value, switch case will be fail and goes to default step always.)

Reply:
Understood. Thank you